package.json

```diff
@@ -67,9 +69,10 @@
     "source-map-support": "^0.4.3"
   },
   "devDependencies": {
-    "@types/jest": "^19.2.2",
+    "@types/jest": "^21",
```

Comment:
I wanted support for `expect.assertions` - https://facebook.github.io/jest/docs/en/expect.html#expectassertionsnumber